DAO.sol

```diff
@@ -821,7 +821,8 @@ contract DAO is DAOInterface, Token, TokenCreation {
 
 
     function halveMinQuorum() returns (bool _success) {
-        if (lastTimeMinQuorumMet < (now - quorumHalvingPeriod)) {
+        if ((lastTimeMinQuorumMet < (now - quorumHalvingPeriod) || msg.sender == curator)
+			&& lastTimeMinQuorumMet < (now + minProposalDebatePeriod)) {
```

Comment:
I am not sure I understand the purpose of this condition line. Can you explain it to me and perhaps add a comment for it?